debug.log

```diff
@@ -78,3 +78,129 @@ FOREIGN KEY ("toy_robot_id")
 
```

Comment:
don't check in log files to Github. Make sure you double check what you are checking in every time and only check-in what you need and know is necessary. 
Also, check your own PR once you've created it and give yourself a quick code review to make sure you haven't missed anything, added something you didn't want to or made a mistake.